src/com/projectkorra/projectkorra/command/HelpCommand.java

```diff
@@ -132,7 +132,7 @@ public void execute(final CommandSender sender, final List<String> args) {
 				sender.sendMessage(ChatColor.WHITE + this.usage + ability.getInstructions());
 			}
 			
-			if (!isPassiveAbility) {
+			if (!isPassiveAbility && (!(sender instanceof Player) || sender.hasPermission("bending.ability." + arg))) {
```

Comment:
Wouldn't it be better to check if they ARE a player? Since they can't bind it as the console anyway

Reply:
Oh wait yeah should have thought about that lol

Reply:
Changed 👍